src/components/Photo/Selfie.tsx

```diff
@@ -130,10 +147,20 @@ export default class SelfieCapture extends Component {
       hasCameraError,
       isCaptureButtonDisabled, // Capture Button is disabled until camera access is allowed + userMedia stream is ready
     } = this.state
+
+    const cameraProps: Omit<CameraProps, 'buttonType'> = {
+      ...this.props,
+    }
+
+    const withTrackingProps: WithTrackingProps = {
+      ...this.props,
+    }
```

Comment:
Same here. I think we can remove those and just use `{...this.props}` below.

Reply:
done